Dockerfile

```diff
@@ -1,3 +1,27 @@
-FROM alpine:latest
+FROM openjdk:8-jdk-slim
```

Comment:
Why don't we just use the `maven:3.6.3-jdk-8-slim` image? That would avoid all of those run steps to download and install it ourselves.

Using that, all you'd need to do is add a `pom.xml` file to this repo, and have a `RUN` step do an `mvn dependency:resolve` to pull down the hiverunner JAR and all of its dependencies?

The `ENTRYPOINT` for this container probably just needs to do an `mvn test`